alacritty/src/event.rs

```diff
@@ -40,21 +39,21 @@ use crate::input::{self, ActionContext as _, Modifiers};
 use crate::window::Window;
 
 #[derive(Copy, Clone, Debug, PartialEq)]
-pub enum FontResize {
+pub enum FontUpdate {
     Delta(f32),
     Reset,
 }
```

Comment:
> I'd assume that Reset is used both for resetting size and for actually changing the font?

That's right; in either case we are resetting the font (size, family, style) on the `Display` and its child structs to the value in the config, whether the event is triggered by:

* a config reload that may include a size, family, or style changes or
* a size reset action (ie whatever is bound to ctrl-0 by default).

> Maybe it makes sense to just remove this option and change the update field type to Option<Font>?

I think this would complicate things a bit since in many `Delta` cases it looks like there is no explicit `Font` available, just an incremental change that needs to be passed to the `handle_update` function (the `f32` parameter on `Delta`).

I'll probably rename `Delta` -> `SizeDelta` in my next set of updates.